data/folders/eos-folder-utilities.directory

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Name=Utilities
```

Comment:
I think we should use `X-GNOME-Utilities.directories` for this one, rather than defining our own. The GNOME one is translated into 78 languages! It's sad that this is the only one we can reuse (Games vs Play, and Coding Education vs our own Learn to Code) but so it goes.

Reply:
OK, I'll update the icon grids and drop this file.